static/js/answers-experience.js

```diff
@@ -0,0 +1,36 @@
+import DeferredPromise from './deferred-promise';
+
+export default class AnswersExperience {
+  constructor (runtimeConfig) {
+    this.runtimeConfig = runtimeConfig;
+    this.AnswersInitializedPromise = new DeferredPromise();
+
+    runtimeConfig.registerListener({
+      eventType: 'update',
```

Comment:
what do you think about not including eventType for now? I think it might make the code slightly simpler, and it's not something we'll be prevented from adding in the future (we can just set the default to 'update' or just refactor).